deployment/prometheus/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM prom/prometheus
```

Comment:
No need to build a new prometheus image. Just mount your config into the container like the existing code does.